runtime/runtime.h

```diff
@@ -153,3 +153,7 @@ void stub_init();
 
 // The runtime entrypoint must be called
 int runtime_main(int argc, char** argv);
+
+extern WEAK u32 wasmg___heap_base;
+extern u32 runtime_heap_base;
```

Comment:
docs why we need these here pls

Reply:
After taking another look, I don't think they were needed. Removed.